sql/src/main/java/com/rilldata/calcite/DependencyFinder.java

```diff
@@ -0,0 +1,53 @@
+package com.rilldata.calcite;
+
+import org.apache.calcite.sql.SqlBasicCall;
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlJoin;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlSelect;
+import org.apache.calcite.sql.ddl.SqlCreateTable;
+import org.apache.calcite.sql.ddl.SqlCreateView;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.sql.util.SqlBasicVisitor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DependencyFinder extends SqlBasicVisitor<List<String>>
+{
+  List<String> dependencies = new ArrayList<>();
+
+  public void visit(SqlNode from1)
```

Comment:
can you rename this method ? this looks like an overriden method of the visitor and looks like will be called upon a `SqlNode`

Reply:
I wonder what name would you like?

Reply:
With the name of the method as visit it seems like it is an overriden method of a visitor class but it's not so suggested name change. Name can be dependencyFinder ?